src/main/java/org/comp2522/ProjectRocketMan/Window.java

```diff
@@ -18,6 +19,8 @@
  *
  */
 public class Window extends PApplet {
```

Comment:
make better Javadoc comments for these changes

Reply:
will do thanks.